agoradatatools/process.py

```diff
@@ -136,27 +134,38 @@ def process_all_files(config_path: str = None, syn=None):
 
     load.create_temp_location(staging_path)
 
+    error_list = []
     if datasets:
         for dataset in datasets:
-            new_syn_tuple = process_dataset(
-                dataset_obj=dataset, staging_path=staging_path, syn=syn
-            )
-            # in the future we should log new_syn_tuples that are none
+            try:
+                new_syn_tuple = process_dataset(
+                    dataset_obj=dataset, staging_path=staging_path, syn=syn
+                )
+                # TODO in the future we should log new_syn_tuples that are none
+            except Exception as e:
+                print(e)
+                error_list.append({list(dataset.keys())[0]: e})
 
     destination = utils._find_config_by_name(config, "destination")
 
-    # create manifest
-    manifest_df = create_data_manifest(parent=destination, syn=syn)
-    manifest_path = load.df_to_csv(
-        df=manifest_df, staging_path=staging_path, filename="data_manifest.csv"
-    )
+    if error_list == []:
```

Comment:
The pep8 standard for checking if a list is empty is:

```
if not error_list:
```

Use the [implicit booleanness](https://docs.python.org/library/stdtypes.html#truth-value-testing).  That said, if you wanted to be explicit, you can also

```
if len(error_list) == 0:
```